README.md

```diff
@@ -1,11 +1,165 @@
 # `apropos-tx`
+Generate a suite of property tests from a logical model.
 
 ## What is this?
+It's a thing that generates property tests for you from a model.
+
+### What kind of model?
+A logical model that describes the device you are testing.
+
+### What device?
+This is the thing you are hooking your test equipment up to.
+
+### What?
+It's a function.
+
+### And how do you write this logical model?
+Good question. First you start with a model. Then you describe it's properties.
```

Comment:
There are some typos here and there in the README. I've fixed some of them in a small PR.

Reply:
Thanks!